changelogs/fragments/53559-docker_swarm_service-mounts-options.yaml

```diff
@@ -0,0 +1,3 @@
+bugfixes:
```

Comment:
That should be `minor_changes:`, as these are no bugfixes :)

Reply:
Thanks, bad copy paste :)